practice_app/pages/index.js

```diff
@@ -94,6 +115,11 @@ export default function Home({ context }) {
                 <h1>Welcome to our demo website!</h1>
                 {country && language && currency && <p>{locationGreeting()} </p>}
                 <MapView onMapClick={onMapClick} coordinates={coordinates} />
+
+                <button onClick={getNumber}>
+                    Click me to get Covid death statistics for the country you have chosen on the map
+                </button>
```

Comment:
This is fine but what do you think about removing the need for a button press, and load the covid stats whenever the user changes location? This can be done by modifying the `onMapClick` function.

Reply:
Honestly i prefer having a front-end part so my code is longer. I barely have enough amount of code so anything that i can add is plus for me.

Reply:
If you want more code on the frontend, my suggestion would be to show the numbers using different colors. If you had implemented my second suggestion about `setCovidStat` earlier, you would have been able to wrap the numbers inside a `<span style={{color: 'red'}}>{data.infected}</span>` to show the infected count in red for example, and recovered in green etc..